### PR TITLE
[Feature] 4K Video

### PR DESCRIPTION
**Pull request by esetnik**
Todo:
- [x] Replaces existing resources with 4K SDR assets
- [ ] Consider allowing user to select both 4K and HD assets
- [x] Fix issue where cached download count is off by 1 (JohnCoates/Aerial/issues/350)
- [x] Fix issue where video would be reused across multiple monitors even after enabling `Play Different Aerial On Each Display`. It is caused because the random video selection does not consider videos already in use. (JohnCoates/Aerial/issues/212)
- [x] Fix issue where titles are cut off (JohnCoates/Aerial/issues/413)

### CODE REVIEW

File: Aerial/Source/Models/ManifestLoader.swift [hunk 1]
```diff
@@ -100,32 +105,31 @@ class ManifestLoader {
...
+                NSLog("Aerial: Encountered unexpected content type for batch")
+                return
+            }
+            
+            let assets = batch["assets"] as! Array<NSDictionary>
             
-            for batch: NSDictionary in batches {
-                let assets = batch["assets"] as! Array<NSDictionary>
+            for item in assets {
+                let url = item["url-4K-SDR"] as! String
```
Comment (mattes) [inline Aerial/Source/Models/ManifestLoader.swift]: What's the difference between the `url-4K-SDR` and `url-4K-HDR` videos?

Comment (abstractpenguin) [reply Aerial/Source/Models/ManifestLoader.swift]: It's likely High Dynamic Range and Standard Dynamic Range. For TVs that support it, HDR has higher contrast range between the brightest and darkest parts of the image.

Comment (mattes) [reply Aerial/Source/Models/ManifestLoader.swift]: Thanks @abstractpenguin!

Found another interesting read: https://github.com/JohnCoates/Aerial/issues/378#issuecomment-333063235

**Comment by timfallmk** [comment]
It looks like that, aside from a testing error, this is ready to merge. Is that correct?

**Comment by fuzzy76** [comment]
Noticed both this merge request and the 4k feature branch seem abandoned, and the issue discussing it has been locked (which I sort of get – that discussion was completely off the rails).

**Comment by esetnik** [comment]
I'm still supporting my feature branch / fork if anyone wants to contribute to it.

**Comment by timfallmk** [comment]
@esetnik Thanks for keeping up with it. What work still needs to be done? I'm happy to help.

**Comment by esetnik** [comment]
None as far as I’m concerned. I’ve been using the 4K aerial for several months without issue.

**Comment by fuzzy76** [comment]
I think @JohnCoates needs to chime in if he wants to use his feature branch or this one for 4k support.

**Comment by scottws2000** [comment]
When will a 4K version come to the iMac Pro?

**Comment by scottws2000** [comment]
@esetnik how do I know if I have your forked version? also thank you for your time and help

**Comment by esetnik** [comment]
@scottws2000 you can test my release [here](https://github.com/esetnik/Aerial/releases/tag/v1.3). Please open issues for any problems you encounter.

**Comment by scottws2000** [comment]
@esetnik Thank you and that looks much better

How do I stay up to date on updates you do?

**Comment by esetnik** [comment]
Closed in favor of https://github.com/JohnCoates/Aerial/pull/501

**Pull request closed without merging**